interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
No, I changed the import_pharmacies.php to show the spinning gif.  I have updated the confirm file now to show the spinning fa fa and ditched the gif.